server/api/core/email/email.js

```diff
@@ -27,27 +26,10 @@ export function send(options) {
  * @returns {Object} returns source
  */
 export function getTemplate(template) {
-  check(template, String);
```

Comment:
The code that was there wasn't actually doing anything yet, so I removed it for the sake of clarity and performance.  That method is called on all of the transactional emails and it contained 3 separate database queries that never actually resulted in a template.  My thinking was that once that functionality actually exists, we could add those parts back.